build/BundledVersions.targets

```diff
@@ -0,0 +1,72 @@
+<Project>
+  <PropertyGroup>
+    <BundledVersionsIntermediateDirectory>$(IntermediateDirectory)/GeneratedMSBuildImports</BundledVersionsIntermediateDirectory>
+    <BundledVersionsPropsFolder>15.0/Imports/Microsoft.Common.props/ImportBefore</BundledVersionsPropsFolder>
+    <BundledVersionsPropsFileName>Microsoft.NETCoreSdk.BundledVersions.props</BundledVersionsPropsFileName>
+  </PropertyGroup>
+
+  <Target Name="GenerateBundledVersionsProps"
+          BeforeTargets="PublishMSBuildExtensions">
+
+    <!-- 
+      The bundled version of Microsoft.NETCore.App is determined statically from DependencyVersions.props whereas the bundled version
+      of NETStandard.Library is defined by what Microsoft.NETCore.App pulls in. This digs in to the package resolution items of
+      of our build against Microsoft.NETCore.App to find the correct NETStandard.Library version
```

Comment:
@ericstj is this dependency staying?

This is equivalent to what dotnet new is currently passing to templates. I didn't want to introduce another thing that can get out sync. If it ever gets removed, it will trip the Error condition and we can decide on how to keep it up to date.